build-deploy/common/models/deployment.js

```diff
@@ -21,35 +21,43 @@ module.exports = function(Deployment) {
 
     debug('deploy to %s: %j', baseURL, deployment);
 
-    resize();
-
-    function deploy(err){
-      if (err) return cb(err);
+    deploy();
 
+    function deploy(){
       if(deployment.type === 'local') {
-        // args are: baseUrl, localdir, config, callback
-        performLocalDeployment(baseURL, cwd, 'process-manager', done);
+        performLocalDeployment(
```

Comment:
Yes, signature changed (cosmetic), they accept a "size:" option now (not cosmetic). You can find the PR that changes them by following up to the arc issue, but I added a "depends on" to the introduction for this PR so its easier.